pkg/cmd/server/admin/create_client.go

```diff
@@ -43,11 +44,11 @@ var createClientLong = templates.LongDesc(`
 	a server certificate authority, and a .kubeconfig file for connecting to the
 	master as the provided user.`)
 
-func NewCommandCreateClient(commandName string, fullName string, out io.Writer) *cobra.Command {
+func NewCommandCreateClient(commandName string, fullName string, streams genericclioptions.IOStreams) *cobra.Command {
 	options := &CreateClientOptions{
 		SignerCertOptions: NewDefaultSignerCertOptions(),
 		ExpireDays:        crypto.DefaultCertificateLifetimeInDays,
-		Output:            out,
+		Output:            streams.Out,
```

Comment:
hm, would it be possible to wire IOStreams through the CreateClientOptions as well?